File: solutions/lua/raindrops/1/raindrops.lua
return function(n)
  local x = ""
  if n % 3 == 0 then
    x = x .. "Pling"
  end
  if n % 5 == 0 then
    x = x .. "Plang"
  end 
  if n % 7 == 0 then
    x = x .. "Plong"
  end
  if x == "" then
    x = tostring(n)
  end
  return x
end

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

func Convert(number int) (result string) {
	s := map[int]string{
		3: "Pling",
		5: "Plang",
		7: "Plong",
	}
	for i, p := range s {
		if number%i == 0 {
			result += p
		}
	}
	if result == "" {
		return strconv.Itoa(number)
	}
	return result
}
